### ISSUE TITLE
Cannot switch item inventory while holding left click

### ISSUE THREAD

**Issue by mish20011** [UnTraDe/terraria-autoswing#3]
I love using this mod when play range but I also use it when I play whips (obsidian w/o feral claws for extra slot), there is a problem though in which when I play with multiple whips to stack attack speed (i.e. Durandal + Dark Harvest + Kaleidoscope) I tried doing it without the mod (w/ feral claws) and I can switch fine, But when I use the mod, I have to stop holding left click for it to switch, 

tl;dr You can only switch items if you stop holding left click in the inventory bar.
